plugins/inventory/vmm_manager.py

```diff
@@ -5,7 +5,9 @@
 from __future__ import (absolute_import, division, print_function)
 import re
 import subprocess
-from subprocess import SubprocessError
+import json
+import os
+from subprocess import PIPE, STDOUT, SubprocessError
```

Comment:
Você já não tem acesso a esses campos pelo `import` da linha 7?
tipo `subprocess.PIPE`
ou eu to doido?

Reply:
Faz sentido. Alterei.